.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-Related Ticket: 
```

Comment:
```suggestion
Related Ticket: 
```
Attempting to incorporate as suggestion, this was removed automatically in vscode